server/coordinator.go

```diff
@@ -520,7 +518,7 @@ func (c *coordinator) removeOperator(op *schedule.Operator) {
 func (c *coordinator) removeOperatorLocked(op *schedule.Operator) {
 	regionID := op.RegionID()
 	delete(c.operators, regionID)
-	c.limiter.UpdateCounts(c.operators)
+	c.limiter.RemoveOperator(op)
```

Comment:
refer to https://github.com/pingcap/pd/pull/874, we meet some not synchronized case so change to` updatecounts`.

Reply:
I have noticed that. But now we have a background goroutine to check all operators, all path will reach RemoveOperator() or AddOperator(). it's unlikely not synchronized.

Reply:
I vote for update limiter using all operators too.